curator/api/utils.py

```diff
@@ -319,11 +320,11 @@ def prune_allocated(client, indices, key, value):
             index=idx,
         )
         try:
-            has_routing = settings[idx]['settings']['index']['routing']['allocation']['require'][key] == value
+            has_routing = settings[idx]['settings']['index']['routing']['allocation'][allocation_type][key] == value
         except KeyError:
             has_routing = False
         if has_routing:
-            logger.debug('Skipping index {0}: Already has allocation rule {1} applied.'.format(idx, key + "=" + value))
+            logger.debug('Skipping index {0}: Already has allocation rule {1} applied for type {2}.'.format(idx, key + "=" + value, allocation_type))
```

Comment:
This doesn't quite fit the syntax of `type.rule=value`.  Perhaps "Allocation rule {1} is already applied for type {2}."  

Thoughts?